src/ui/server/routes/wellknown/index.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { ROUTES } from '../../constants';
+import get from '../../controllers/wellknown/security';
+
+// @ts-ignore
+const wellknownRouter = express.Router();
+
+// Routes
+wellknownRouter.get(ROUTES.WELL_KNOWN.SECURITY, get);
```

Comment:
Can we create a unit test please like the other routes

Also no need for the `Routes` comment IMO, self explanatory